src/mqttcm_connect.c

```diff
@@ -464,7 +470,14 @@ void on_subscribe(struct mosquitto *mosq, void *obj, int mid, int qos_count, con
         //SUBSCRIBE can contain many topics at once
 
 	//send on_subscribe callback event to webconfig via rbus.
-	sendRbusEventWebcfgOnSubscribe();
+	MqttCMInfo("mid for sub is %d\n", mid);
+	char *topicname = GetTopicFromSubcribeId(mid);
```

Comment:
Add NULL check for topicname and log, also add else case to log an error "failed to get subscribe topic name from mid"